Controller/MediaAdminController.php

```diff
@@ -108,4 +115,101 @@ public function listAction(Request $request = null)
             'csrf_token' => $this->getCsrfToken('sonata.batch'),
         ));
     }
+
+    /**
+     * @throws \Symfony\Component\Security\Core\Exception\AccessDeniedException
+     *
+     * @return \Symfony\Bundle\FrameworkBundle\Controller\Response|\Symfony\Component\HttpFoundation\Response
+     */
+    public function multiUploadAction()
+    {
+        if (false === $this->admin->isGranted('CREATE')) {
```

Comment:
`isGranted` always returns a boolean, which means you can simply write `!$this->admin->isGranted('CREATE')`